web/templ/components/base/sidenav.templ

```diff
@@ -10,7 +10,7 @@ templ SideNav(page string) {
 		hx-boost="true"
 		hx-target="#sidenav-page-wrapper"
 		hx-swap="outerHTML"
-		class="fixed -right-[--sidenav-width] md:right-0 top-16 p-2
+		class="fixed -right-[--sidenav-width] md:right-0 top-16 p-2 overflow-y-auto
```

Comment:
I noticed that after fixing the strict HTML problem in a previous PR, it broke the overflow scrolling of the sidenav bar. This resolves this.